REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 1.0
```

Comment:
Afaict, Coveralls still fails on 0.7, I assume due to a versioning issue. Should this be changed to `0.7`?